kivymd/uix/bottomnavigation.py

```diff
@@ -274,7 +274,7 @@ def build(self):
             text_color: root._text_color_normal
             valign: 'middle'
             halign: 'center'
-            opposite_colors: root.opposite_colors
+            opposite_colors: root.opposite_colors if root.opposite_colors is not None else True
             pos: [self.pos[0], self.pos[1]]
             font_size: dp(24)
             pos_hint: {'center_x': .5, 'center_y': .7}
```

Comment:
this will avoid any error when using opposite colors with labels, since it's a value required to work now.